src/lib.rs

```diff
@@ -93,6 +93,9 @@ mod dns;
 mod kdc;
 mod krb;
 mod secret;
+#[cfg(feature = "scard")]
+#[allow(dead_code)]
```

Comment:
it's a temporary code. just to pass the CI and avoid warnings about unused code

Reply:
Sounds good :+1: